File: 101/exercise_rework/easy_9/grade_book.rb
# grade_book.rb

def get_grade(v, v1, v2)
  mean = (v + v1 + v2)/3
  if (0...60).cover?(mean)
    'F'
  elsif (60...70).cover?(mean)
    'D'
  elsif (70...80).cover?(mean)
    'C'
  elsif (80...90).cover?(mean)
    'B'
  elsif (90...100).cover?(mean)
    'A'
  else
    'NA'
  end
end

File: 210/lesson_3/logic_and_flow/small_problems/easy2/grade_book.js
// grade_book.js

function getGrade(score, score1, score2) {
  var avg = getAvg(score, score1, score2);
  var letterGrade;
  if (avg >= 90) {
    letterGrade = 'A';
  } else if (avg < 90 && avg >= 80) {
    letterGrade = 'B';
  } else if (avg < 80 && avg >= 70) {
    letterGrade = 'C';
  } else if (avg < 70 && avg >= 60) {
    letterGrade = 'D';
  } else {
    letterGrade = 'F';
  }
  return letterGrade;
}

function getAvg(score, score1, score2) {
  return (score + score1 + score2) / 3;
}

console.log(getGrade(95, 90, 93));    // "A"
console.log(getGrade(50, 50, 95));    // "D"